package.json

```diff
@@ -80,7 +80,13 @@
   "devDependencies": {
     "@11ty/eleventy": "^0.10.0",
     "@11ty/eleventy-plugin-inclusive-language": "^1.0.0",
+    "@babel/preset-env": "^7.10.2",
     "@mocha/docdash": "^2.1.3",
+    "@rollup/plugin-babel": "^5.0.3",
+    "@rollup/plugin-commonjs": "^11.0.2",
+    "@rollup/plugin-json": "^4.0.2",
+    "@rollup/plugin-multi-entry": "^3.0.1",
+    "@rollup/plugin-node-resolve": "^7.1.1",
     "assetgraph-builder": "^8.0.1",
     "autoprefixer": "^9.7.4",
     "babel-eslint": "^10.1.0",
```

Comment:
Do we still need _browserify_ and _browserify-package-json_ devDependencies?
And below the `browserify` property?

Reply:
I forgot to clean up properly. Fixed in c052f96a